spec/integration/outputs/pipeline_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative "../../../spec/es_spec_helper"
+
+describe "Ingest pipeline execution behavior", :integration => true do
```

Comment:
Hmm you only want to run this test on > 5.0.0. So maybe check `ES_VERSION=5.0.0-alpha1` here or add another rspec tag